locust/dispatch.py

```diff
@@ -163,13 +163,16 @@ def _dispatcher(self) -> Generator[Dict[str, Dict[str, int]], None, None]:
 
         self._dispatch_in_progress = False
 
-    def new_dispatch(self, target_user_count: int, spawn_rate: float) -> None:
+    def new_dispatch(self, target_user_count: int, spawn_rate: float, user_classes: any) -> None:
```

Comment:
`any` is unnecessarily broad, isnt it?

Reply:
(something like Optional[List[User]])

Reply:
Yeah, you're totally right. I did this for the small test on my system and forgot to change it. 
I changed also all the other type hints according to that.